Makefile

```diff
@@ -65,12 +65,12 @@ black: check_venv
 install: venv
 	( . venv/bin/activate && pip install -U pip && pip install -r requirements.txt && python setup.py develop && pre-commit install )
 
+install-docs: venv
+	( . venv/bin/activate && pip install -r docs-requirements.txt )
```

Comment:
I think the more common (modern) location for the requirements file needed for the docs is in the `docs/` directory. (Goes along with a separate Makefile in the docs directory.)  But this is certainly not a blocker.

Nit:: I believe the more common file name is `requirements-docs.txt` (4,074 on GitHub, 12 in Mozilla) `doc-requirements.txt` is less (1913, 0 in Mozilla), and `docs-requiements.txt` doesn't appear.

Reply:
I moved the file to `docs/requirements.txt`. In the `docs` directory, I don't need any further qualification for the name.